firestore/runtime/src/main/java/io/quarkiverse/googlecloudservices/firestore/runtime/FirestoreProducer.java

```diff
@@ -8,6 +8,7 @@
 import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
 
+import org.eclipse.microprofile.config.inject.ConfigProperty;
```

Comment:
All the changes on this class should not be needed if using the default quarkus.google.cloud.firestore.host-override property.